packages/desktop-gui/src/auth/login-form.jsx

```diff
@@ -20,25 +21,27 @@ class LoginForm extends Component {
 
   render () {
     const { message } = authStore
+    const { buttonClassName } = this.props
 
     return (
       <div className='login-content'>
         {this._error()}
-        <button
-          className={cs('btn btn-login btn-primary btn-wide', {
-            disabled: this.state.isLoggingIn,
-          })}
-          onClick={this._login}
-          disabled={this.state.isLoggingIn}
-        >
-          {this._buttonContent(message)}
-        </button>
+        <div className='button-wrapper'>
+          <button
+            className={cs(buttonClassName, {
+              disabled: this.state.isLoggingIn,
+            })}
+            onClick={this._login}
+            disabled={this.state.isLoggingIn}
+          >
+            {this._buttonContent(message)}
+          </button>
+        </div>
         {
           message && <p className={`message ${message.type}`} onClick={this._selectUrl}>
             <MarkdownRenderer markdown={message.message}/>
           </p>
         }
-        <p className='terms'>By logging in, you agree to the <a onClick={this._openTerms}>Terms of Use</a> and <a onClick={this._openPrivacy}>Privacy Policy</a>.</p>
```

Comment:
What's the rationale behind removing the acceptance of Terms and Privacy Policy?

Reply:
So during our design sessions with Brian he said this is no longer needed since logging in is done in the browser (where that message is evident). This snuck into this PR since we were originally going to use this component to directly log in from prompts but that behavior has since been changed.